index.html

```diff
@@ -16,7 +16,14 @@
   <header>
     <a class="log">My Logo</a>
     <nav>
```

Comment:
Good job! but you need to implement some minor changes, don't worry I'll explain all of them simply.

As you know it's required to follow the exact Figma template, so please make sure to adapt your design to the Figma template.
1. It's required to have a space from left and right in almost all parts of your desktop page.
2. [Optional] Don't you think it'll be a good idea to add a `container` class and use it everywhere to have the same left and right padding for all of your layout contents?
3. Please make sure to use the exact colors as the ones in the Figma design.

![image](https://user-images.githubusercontent.com/34779928/194391495-6ad5dcfd-3e06-47f0-ba15-cdb4feeefefc.png)


![image](https://user-images.githubusercontent.com/34779928/194390974-117ab58a-8bb9-430f-bce0-17728a0bf773.png)